tests/test_registrations/test_views.py

```diff
@@ -0,0 +1,476 @@
+        assert_equal(res.status_code, http.OK)
+
+        open_ended_schema = MetaSchema.find_one(
+            Q('name', 'eq', 'OSF-Standard Pre-Data Collection Registration') &
+            Q('schema_version', 'eq', 1)
+        )
+        self.draft.reload()
+        assert_equal(open_ended_schema, self.draft.registration_schema)
+        assert_equal(metadata, self.draft.registration_metadata)
+
+    def test_update_draft_registration_non_admin(self):
+        metadata = {
+            'summary': {
+                'value': 'updated',
+                'comments': []
+            }
+        }
+        assert_not_equal(metadata, self.draft.registration_metadata)
+        payload = {
+            'schema_data': metadata,
+            'schema_name': 'OSF-Standard Pre-Data Collection Registration',
+            'schema_version': 1
+        }
+        url = self.node.api_url_for('update_draft_registration', draft_id=self.draft._id)
+
+        res = self.app.put_json(url, payload, auth=self.non_admin.auth, expect_errors=True)
+        assert_equal(res.status_code, http.FORBIDDEN)
+
+    def test_delete_draft_registration(self):
+        assert_equal(1, DraftRegistration.find().count())
+        url = self.node.api_url_for('delete_draft_registration', draft_id=self.draft._id)
+
+        res = self.app.delete(url, auth=self.user.auth)
+        assert_equal(res.status_code, http.NO_CONTENT)
+        assert_equal(0, DraftRegistration.find().count())
+
+    def test_delete_draft_registration_non_admin(self):
+        assert_equal(1, DraftRegistration.find().count())
+        url = self.node.api_url_for('delete_draft_registration', draft_id=self.draft._id)
+
+        res = self.app.delete(url, auth=self.non_admin.auth, expect_errors=True)
+        assert_equal(res.status_code, http.FORBIDDEN)
+        assert_equal(1, DraftRegistration.find().count())
+
+    @mock.patch('website.archiver.tasks.archive')
+    def test_delete_draft_registration_registered(self, mock_register_draft):
+        self.draft.register(auth=Auth(self.user), save=True)
+        url = self.node.api_url_for('delete_draft_registration', draft_id=self.draft._id)
+
+        res = self.app.delete(url, auth=self.user.auth, expect_errors=True)
+        assert_equal(res.status_code, http.FORBIDDEN)
+
+    def test_only_admin_can_delete_registration(self):
+        non_admin = AuthUserFactory()
+        assert_equal(1, DraftRegistration.find().count())
+        url = self.node.api_url_for('delete_draft_registration', draft_id=self.draft._id)
+
+        res = self.app.delete(url, auth=non_admin.auth, expect_errors=True)
+        assert_equal(res.status_code, http.FORBIDDEN)
+        assert_equal(1, DraftRegistration.find().count())
+
+    def test_get_metaschemas(self):
+        url = api_url_for('get_metaschemas')
+        res = self.app.get(url).json
+        assert_equal(len(res['meta_schemas']), len(ACTIVE_META_SCHEMAS))
+
+        url = '/api/v1/project/drafts/schemas/?include=all'
```

Comment:
Other than the addition of `include=all`, this URL seems to be identical to the above. It's a good idea to avoid hard-coded URLs in tests, but doubly so if the hard-coded URL appears to be a duplicate request.